tests/test-dirs/refactor-open/unqualify.t

```diff
@@ -0,0 +1,136 @@
+Works for in-file modules
+  $ $MERLIN single refactor-open -action unqualify -position 4:6 <<EOF
+  > module M = struct
+  >   let u = ()
+  > end
+  > open M
+  > let u = M.u
+  > EOF
+  {
+    "class": "return",
+    "value": [
+      {
+        "start": {
+          "line": 5,
+          "col": 8
+        },
+        "end": {
+          "line": 5,
+          "col": 11
+        },
+        "content": "u"
+      }
+    ],
+    "notifications": []
+  }
+
+Works for in-file nested modules
+
+  $ $MERLIN single refactor-open -action unqualify -position 6:6 <<EOF
+  > module M = struct
+  >   module N = struct
+  >     let u = ()
+  >   end
+  > end
+  > open M.N
+  > let u = M.N.u
+  > EOF
+  {
+    "class": "return",
+    "value": [
+      {
+        "start": {
+          "line": 7,
+          "col": 8
+        },
+        "end": {
+          "line": 7,
+          "col": 13
+        },
+        "content": "u"
+      }
+    ],
+    "notifications": []
+  }
+
+Works for stdlib modules (stdlib modules differ from other in-file modules because their
+full path is different)
```

Comment:
Not sure what you're trying to test, but Unix is not part of the stdlib.

(Repeated from above, because one of the two will probably disappear after a rebase?)

Reply:
Still not sure what you're trying to test. This test already behaves like that before the PR.

Reply:
I wanted to make sure unqualify works the same for the modules defined in the file where unqualify is called and for modules defined outside of that file. I put the test to show that the PR doesn't bring unintended changes 

If that doesn't make sense, I can remove the test.

Reply:
Yes, I don't think that test makes sense.